### PR TITLE
PROP-0003 -- BitKarma

### PR DESCRIPTION
**Pull request by derbund**
The idea is to have a simple first product for Gitcorp that the community can CRUSH quickly and start seeing revenue right away 🤑. I didn't list competitors because I don't feel it's relevant to whether or not we do this idea: we're not trying to win the market, just getting our feet wet as a community with product development.

### CODE REVIEW

**Comment by kvh** [comment]
I like the idea of a bite-sized project to start, although how bite sized is this?

File: planning/proposals/props/prop-0003.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+<img width="80" src="prop-0003/bitkarma.png">
+
+A simple embed for any project, app, or organization to instantly and securely
+accept Bitcoin donations. The end-user sees a send address and gets fast,
+visible confirmation inside the same site or app that their donation has gone
+through and they are now a proud supporter, providing instant gratification and
+reinforcing the donation behavior.
+
+
```
Comment (kvh) [inline planning/proposals/props/prop-0003.md]: Remove extra line

File: planning/proposals/props/prop-0003.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+### Business model
+
+Take a small 1% transaction fee off of every donation.
+
+### Why Gitcorp?
+
+A small, easily deliverable project that seems like a great first product for
+Gitcorp to get its feet wet with. Also, it's a project Gitcorp itself could use!
+
+### Existing solutions
```
Comment (kvh) [inline planning/proposals/props/prop-0003.md]: Just kill this section if you are not going to use it.

File: planning/proposals/props/prop-0003.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+</pre>
+
+
+## BitKarma
+
+#### Bitcoin donation button and embed
+
+### Problem
+
+Open source projects are always looking for new ways to support themselves, but
```
Comment (kvh) [inline planning/proposals/props/prop-0003.md]: This problem exists for a lot more than just open source projects; any independent creator -- blogger, artist, etc -- could benefit from the solution. Ok to limit the target market, but expand the problem statement to reflect the whole problem.

File: planning/proposals/props/prop-0003.md (hunk 2 above)
Comment (kvh) [reply planning/proposals/props/prop-0003.md]: Or, you know, actually fill it out 😁. Patreon, paypal donate, etc.

Comment (derbund) [reply planning/proposals/props/prop-0003.md]: Yeah, yeah, yeah... done :)

**Pull request merged**